tortoise/tortoise.go

```diff
@@ -761,8 +761,12 @@ func (t *turtle) addLocalVotes(ctx context.Context, logger log.Log, lid types.La
 			t.undecided[lid] = struct{}{}
 			return nil
 		}
-		for _, bid := range hareOutput {
-			t.hareOutput[bid] = support
+		bids := t.blocks[lid]
+		for _, bid := range bids {
```

Comment:
please remove this loop. if hare doesn't support a block it should not be added to the hareOutput